aerospike/assets/dashboards/overview.json

```diff
@@ -1,960 +1,707 @@
 {
-    "title": "Aerospike Overview",
-    "description": "## Aerospike Dashboard\n\nThis is an example Aerospike dashboard demonstrating the metrics that the integration collects.",
+    "author_name": "Datadog",
+    "description": "## Aerospike Dashboard\n\nDatadog's integration for Aerospike gives you the visibility you need to identify performance issues and monitor the failovers and events.\n\nThis dashboard shows an overview of your Aerospike infrastructure, and provides more information about records or queries.\n\nClone this template dashboard to make changes and add your own graph widgets.",
+    "layout_type": "free",
+    "template_variables": [
+        {
+            "default": "*",
+            "name": "host",
+            "prefix": "host"
+        },
+        {
+            "default": "*",
+            "name": "namespace",
+            "prefix": "namespace"
+        }
+    ],
+    "title": "Aerospike - Overview",
```

Comment:
Is the title change intentional? Syncing this will require updating pg manually

Reply:
I'm changing it back.

AFAIK, changing the title is fine. A manual pg update is only needed when the `short name` is update (in `manifest.json`, which is not the case.